### PR TITLE
Added momentum to optimization algorithms

### PR DESCRIPTION
**Pull request by pablovela5620**
First time contributing to an opensource project! Let me know if there's anything I need to fix. I hope to contribute more in the future

### CODE REVIEW

File: docs/optimizers.rst [hunk 1]
```diff
@@ -41,8 +41,23 @@ Be the first to `contribute! <https://github.com/bfortuner/ml-cheatsheet>`__
...
+.. math::
+
+
+    v_{dW} = \beta v_{dW} + (1 - \beta) dW \\
+    W = W - \alpha v_{dW}
+
+.. note::
 
+  - :math:`v` - the exponentially weighted average
+  - :math:`dW` - cost graident with respect to current layer weight tensor
```
Comment (bfortuner) [inline docs/optimizers.rst]: graident should be "gradient"

**Comment by bfortuner** [comment]
Looks great! Small typo in "gradient" and maybe missing a period in the last sentence of the intro.

Will merge once fix or I can fix later if you're short for time.

**Comment by pablovela5620** [comment]
Should be fixed!

**Comment by bfortuner** [comment]
Awesome! Look forward to working with you :)

**Pull request merged**